doc/src/modules/physics/mechanics/examples/multi_degree_freedom_holonomic_system.rst

```diff
@@ -0,0 +1,87 @@
+    >>> simple_pend.apply_torque(kT*q3*simple_pend.z, reaction_body=compound_pend)
+    >>> block.apply_force(-wall.y*block.mass*g)
+    >>> compound_pend.apply_force(-wall.y*compound_pend.mass*g)
+    >>> simple_pend.apply_force(-wall.y*simple_pend.mass*g)
+
+With the problem setup, the equations of motion can be generated using the
+``JointsMethod`` class with KanesMethod in backend. ::
+
+    >>> method = JointsMethod(wall, slider, rev1, rev2)
+    >>> soln = method.form_eoms()
+
+    >>> method.mass_matrix_full
+    Matrix([
+    [1, 0, 0,                                                                        0,                                         0,                                                                        0],
+    [0, 1, 0,                                                                        0,                                         0,                                                                        0],
+    [0, 0, 1,                                                                        0,                                         0,                                                                        0],
+    [0, 0, 0,                                                             ma + mb + mc, 2*l*mb*cos(q2(t))/3 + 2*l*mc*cos(q2(t))/3, mc*(l*(-sin(q2(t))*sin(q3(t)) + cos(q2(t))*cos(q3(t))) + l*cos(q2(t))/3)],
+    [0, 0, 0,                                2*l*mb*cos(q2(t))/3 + 2*l*mc*cos(q2(t))/3,          IBzz + 4*l**2*mb/9 + 4*l**2*mc/9,                                      mc*(2*l**2*cos(q3(t))/3 + 2*l**2/9)],
+    [0, 0, 0, mc*(l*(-sin(q2(t))*sin(q3(t)) + cos(q2(t))*cos(q3(t))) + l*cos(q2(t))/3),       mc*(2*l**2*cos(q3(t))/3 + 2*l**2/9),                                     mc*(2*l**2*cos(q3(t))/3 + 10*l**2/9)]])
+
+    >>> method.forcing_full
+    Matrix([
+    [                                                                                                                                                                                  u1(t)],
+    [                                                                                                                                                                                  u2(t)],
+    [                                                                                                                                                                                  u3(t)],
+    [ -c*u1(t) - k*q1(t) + 2*l*mb*u2(t)**2*sin(q2(t))/3 - l*mc*(-sin(q2(t))*cos(q3(t)) - sin(q3(t))*cos(q2(t)))*(u2(t) + u3(t))*u3(t) + mc*(2*l*u2(t)/3 + l*u3(t)/3)*u2(t)*sin(q2(t)) + F(t)],
+    [                                                                                   -2*g*l*mb*sin(q2(t))/3 - 2*g*l*mc*sin(q2(t))/3 + 2*l**2*mc*(u2(t) + u3(t))*u3(t)*sin(q3(t))/3 + T(t)],
+    [-g*l*mc*(sin(q2(t))*cos(q3(t)) + sin(q3(t))*cos(q2(t))) - g*l*mc*sin(q2(t))/3 + kT*q3(t) + l**2*mc*(u2(t) + u3(t))*u3(t)*sin(q3(t))/3 - l*mc*(2*l*u2(t)/3 + l*u3(t)/3)*u2(t)*sin(q3(t))]])
```

Comment:
Make sure these eoms are the exact same as my result in the pydy example.

Reply:
Checked